common/djangoapps/student/tests/tests.py

```diff
@@ -930,3 +932,190 @@ def test_roundtrip_with_unicode_course_id(self):
         real_user = user_by_anonymous_id(anonymous_id)
         self.assertEqual(self.user, real_user)
         self.assertEqual(anonymous_id, anonymous_id_for_user(self.user, course2.id, save=False))
+
+
+@ddt.ddt
+class DashboardTestXSeriesPrograms(ModuleStoreTestCase):
```

Comment:
Also - if it is compatible - please inherit from `SharedModuleStoreTestCase` here, which is much faster (see: http://swampcastle.org/2015/07/28/writing-faster-modulestore-tests.html)